### ISSUE TITLE
EPANET_MTP4r2_ Standard-Toolbar_Create-time-series_High

### ISSUE THREAD

**Issue by rjanke20** [USEPA/SWMM-EPANET_User_Interface#227]
Water quality simulation for 168 hrs reporting time step results in tabular results for only 27 hours.  Also, left most column heading in tabular time series table should be "Time Hours".

Also, if memory is empty provides - “Model status not found. Run the model to generate model status.”  HOWEVER, IF PREVIOUSLY RUN MODEL, STATUS WILL BE PROVIDED WHICH COULD BE WRONG.

**Comment by PaulDudaRESPEC** [comment]
I ran some tests and each time it gives me the number of reporting time steps that I expect.  The Heading of the left most column is not something the user can set using the QTableWidget; it's probably apparent that those are times since the user selected 'Time Series' but if not we can add something.

How to handle clearing the model status is something we should consider and address.

**Comment by TongZhai** [comment]
@rjanke20 **The input file for this issue is needed for us to reproduce the truncated tabular result that you observed.** We have not been able to reproduce this behavior. 

The top left corner cell is actually a button for selecting all cells' content in a table. It is not editable. With PyQt, the workaround is to remove the top default header row and instead use the first data row as the header row and then manage all UI event by ourselves. I am not sure this is necessary at this point. 

About the clearing of status report from a previous run, it has to do with the ability to detect whether a model has been changed since last save, I've created a new feature request on the issue list as a later date effort.
